tests/network-tests/src/flows/contentDirectory/creatingChannel.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryNodeApi } from '../../Api'
+import { CreateChannelFixture } from '../../fixtures/contentDirectoryModule'
+import { ChannelEntity } from 'cd-schemas/types/entities/ChannelEntity'
+import { assert } from 'chai'
+import { KeyringPair } from '@polkadot/keyring/types'
+
+export function createSimpleChannelFixture(api: QueryNodeApi, pair: KeyringPair): CreateChannelFixture {
+  const channelEntity: ChannelEntity = {
+    title: 'Example channel',
+    description: 'This is an example channel',
+    // We can use "existing" syntax to reference either an on-chain entity or other entity that's part of the same batch.
+    // Here we reference language that we assume was added by initialization script (initialize:dev), as it is part of
+    // input/entityBatches/LanguageBatch.json
+    language: { existing: { code: 'EN' } },
+    coverPhotoUrl: '',
+    avatarPhotoURL: '',
+    isPublic: true,
+  }
+  return new CreateChannelFixture(api, channelEntity, pair)
+}
+
+async function delay(ms: number) {
+  await new Promise((resolve) => setTimeout(() => resolve(), ms)).then(() => console.log('fired'))
+}
+
+export default async function channelCreation(api: QueryNodeApi, pair: KeyringPair) {
+  const createChannelHappyCaseFixture = createSimpleChannelFixture(api, pair)
+
+  await createChannelHappyCaseFixture.runner(false)
+
+  // Temporary solution (wait 10 minutes)
+  await delay(600000)
+
+  await api
+    .getChannelbyTitle(createChannelHappyCaseFixture.channelEntity.title)
+    .then((result) => console.log(result.data))
```

Comment:
I can never make sense of mixing `async` and `.then` I suggest to try below instead and see if you get the output you expect

```suggestion
  const result = await api.getChannelbyTitle(createChannelHappyCaseFixture.channelEntity.title)
  console.log(result.data)
```

Reply:
Done https://github.com/Joystream/joystream/pull/1565/commits/68ecab08a3686833e4299d44446875efedac3740#diff-ea5a308b86eec0764783d35832f37f7178e6bd7a6da02ab8c47f1354058e425aR34